### PR TITLE
style: change in ExportButton style (#6147)

### PR DESCRIPTION
**Pull request by Nishant-l**
IMG_334945D5F47E-1

### CODE REVIEW

**Comment by dwelle** [comment]
Hello @Nishant-l — what is this solving?

**Comment by Nishant-l** [comment]
Hi @dwelle 

- On ipad the the text on the PNG and SVG buttons are not centred it looks like this

IMG_09FAD9E16219-1

- And after this fix they are centred and looks like this

IMG_334945D5F47E-1

File: src/components/ExportDialog.scss [hunk 1]
```diff
@@ -96,6 +96,7 @@
     width: 5rem;
     height: 5rem;
     margin: 0 0.2em;
+    padding: 0em;
```
Comment (dwelle) [inline src/components/ExportDialog.scss:99]:
```suggestion
    padding: 0em;
    display: flex;
    align-items: center;
    justify-content: center;
```

Comment (dwelle) [reply src/components/ExportDialog.scss:99]: this will also improve positioning of the copy button icon

**Comment by dwelle** [comment]
I see, interesting! I've made one tweak to that: https://github.com/excalidraw/excalidraw/pull/6148#discussion_r1083442492

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
Thanks for the fix! ❤️

**Pull request merged**